GenericsHomework/GenericsHomeworkTests/NodeTests.cs

```diff
@@ -0,0 +1,56 @@
+using GenericsHomework;
+using System;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+
+namespace GenericsHomeworkTests;
+
+[TestClass]
+public class NodeTests
+{
+    public Node<object>? _Node;
+
+    [TestInitialize]
+    public void Init()
+    {
+        Node<object> nn = new Node<object>(1);
+        _Node = nn;
+    }
+
+    [TestMethod]
+    public void TestToStringReturnsCorrectly()
+    {
+        Assert.AreEqual<object>(_Node!.ToString(), "1");
```

Comment:
```suggestion
        Assert.AreEqual<string>(_Node!.ToString(), "1");
```
Using `object` sort of defeats the point of generics.